kqueen_ui/blueprints/ui/templates/ui/organization_manage.html

```diff
@@ -14,20 +14,26 @@ <h3>Members</h3>
   </div>
   <table class="table table-hover">
     <thead>
-      <th class="col-md-3">Name</th>
-      <th class="col-md-3">E-mail</th>
-      <th class="col-md-2">Role</th>
-      <th class="col-md-4">Created</th>
-      <th class="status_column">Status</th>
-      <th class="action_column">Actions</th>
+      <tr>
+        <th class="col-md-3">Name</th>
+        <th class="col-md-3">E-mail</th>
+        <th class="col-md-2">Role</th>
+        <th class="col-md-4">Created</th>
+        <th class="status_column">Status</th>
+        <th class="action_column">Actions</th>
+      </tr>
     </thead>
     <tbody>
       {% for member in members %}
         <tr>
           <td class="col-md-3">{{ member.username }}</td>
           <td class="col-md-3">{{ member.email }}</td>
           <td class="col-md-2">{{ member.role }}</td>
-          <td class="col-md-4">{{ member.created_at }}</td>
+          <td class="col-md-4">
```

Comment:
it is not looks ok to use script tag at html files at all, I suggest to add class and  perform formatDate to the content everywhere that class is presented

Reply:
Yes, but if we replace it with jQuery after render, then in case of large table user will see how content is changing..